lib/active_admin/mongoid/inputs/filters/select_input.rb

```diff
@@ -0,0 +1,12 @@
+require 'active_admin/inputs/filters/select_input'
+
+class ActiveAdmin::Inputs::Filters::SelectInput
+  def searchable_method_name
+    name = if searchable_has_many_through?
+      "#{reflection.through_reflection.name}_#{reflection.foreign_key}"
+    else
+      reflection.key if reflection_searchable?
```

Comment:
I feel like it would be better if that `if oneliner` was actually changed into `elsif` as you did it [here](https://github.com/activeadmin/activeadmin-mongoid/pull/125/files#diff-446008629bce473ff0cfbb647f4c6c28R9).